tests/javascript/position_caret/test_position_caret.py

```diff
@@ -0,0 +1,94 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2015 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Tests for position_caret.js."""
+
+import pytest
+
+from PyQt5.QtCore import Qt
+from PyQt5.QtWebKit import QWebSettings
+from PyQt5.QtWebKitWidgets import QWebPage
+
+
+@pytest.fixture(autouse=True)
+def enable_caret_browsing():
+    """Fixture to enable caret browsing globally."""
```

Comment:
It probably doesn't matter (at least now, it might always bite me later :wink:), but I agree it's cleaner this way, so I added that.